tensorflow/python/training/saver.py

```diff
@@ -1070,6 +1070,10 @@ def save(self, sess, save_path, global_step=None, latest_filename=None,
     if not isinstance(sess, session.SessionInterface):
       raise TypeError("'sess' must be a Session; %s" % sess)
 
+    # Note a few lines above save_path was set to os.path.dirname(save_path)
```

Comment:
save_file = 'model.ckpt'

When I try to do 

```
tf.saver = tf.train.Saver()
with tf.Session() as sess:
     # Do something 
     saver.save(sess, save_file)
```

I get this error: 
```
C:\Program Files\Anaconda3\envs\IntroToTensorFlow\lib\site-packages\tensorflow\python\training\saver.py in save(self, sess, save_path, global_step, latest_filename, meta_graph_suffix, write_meta_graph, write_state)
   1312     if not gfile.IsDirectory(os.path.dirname(save_path)):
   1313       raise ValueError(
-> 1314           "Parent directory of {} doesn't exist, can't save.".format(save_path))
   1315 
   1316     save_path = os.path.dirname(save_path)

ValueError: Parent directory of model.ckpt doesn't exist, can't save.
```

I am using tensorflow 0.12.0 in ancaconda 3 using windows 8.